c_binding/include/libstoragemgmt/libstoragemgmt_types.h

```diff
@@ -378,6 +383,35 @@ typedef enum {
 #define LSM_SYSTEM_CACHE_PCT_NO_SUPPORT             -2
 #define LSM_SYSTEM_CACHE_PCT_UNKNOWN                -1
 
+typedef enum {
+    LSM_BATTERY_TYPE_UNKNOWN = 1,
+    LSM_BATTERY_TYPE_OTHER = 2,
+    LSM_BATTERY_TYPE_CHEMICAL = 3,
+    LSM_BATTERY_TYPE_CAPACITOR = 4,
+} lsm_battery_type;
```

Comment:
Right above this `lsm_system_mode_type` is used for `LSM_SYSTEM_MODE_...`  when `lsm_system_mode` would do. This seems to indicate you use the `_type` suffix to denote typedef-ed enums. But I also see that `lsm_job_status` is present, which doesn't do this. Looking through the code again, I see that the other `_type` names are appropriate. I think I interpreted incorrectly. Please disregard my original comment.